engine/server/engine/main.go

```diff
@@ -411,14 +413,15 @@ func restApiServer(
 
 	// This is how you set up a basic Echo router
 	echoRouter := echo.New()
-	echoRouter.Use(echomiddleware.Logger())
+	echoApiRouter := echoRouter.Group("/api")
```

Comment:
I think you can use this constant here `pathToApiGroup`

Reply:
fixed. nice catch!